### ISSUE TITLE
l10n: There is no text string in Transifex

### ISSUE THREAD

**Issue by Valdnet** [nextcloud/calendar#3441]
The following text string is missing from Transifex:

https://github.com/nextcloud/calendar/blob/9697327072cef227f1c414c0764b287402ef66a7/src/components/Editor/Repeat/RepeatFirstLastSelect.vue#L65

https://github.com/nextcloud/calendar/blob/2d5f39bed56c217ba1421b91323ca5860a6e5ee2/src/filters/recurrenceRuleFormat.js#L208

![obraz](https://user-images.githubusercontent.com/47037905/131476308-8b30122d-4bd0-4469-b13a-7242bd4561f1.png)

This is because a similar text string is found in this:

https://github.com/nextcloud/calendar/blob/ecac7cc447b868d5c3d7b668acc8d7cf139b3513/src/components/Editor/Alarm/AlarmTimeUnitSelect.vue#L74

![obraz](https://user-images.githubusercontent.com/47037905/131477526-536ffc52-2c71-47fa-9f6d-790a8ce00919.png)

which is in the plural.

@ChristophWurst Do you have any idea for that?

**Comment by tcitworld** [comment]
I think we already had the issue once.

**Comment by rakekniven** [comment]
Still the same for v3.0.1 on NC v22.2.3
Browser: Firefox v94
OS: macOS Big Sur 11.6.1

**Comment by rakekniven** [comment]
Thank your digging into it.

Unfortunately I have no good idea for another wording.

**Comment by SebastianKrupinski** [comment]
@rakekniven we are probably going to need another class for positions "this.$t('calendar-poistion', 'second')"

Let me check in to this.

**Comment by SebastianKrupinski** [comment]
@rakekniven I have not seen the other tickets, can you link them here, if you have them on hand.

**Comment by st3iny** [comment]
I implemented a whacky solution at #6752.

**Comment by st3iny** [comment]
> Do I understand correctly that you have changed the plural function to pure plural?

Yes, the plural is now always used in the reminder select. Now, two distinct strings are registered which should be translatable independently of each other, i.e., `second` and `seconds`.

**Comment by st3iny** [comment]
Hmm, I just realized that we will need to add context now. The first term `second` refers to the second event in a series but the second term `seconds` refers to a time unit. That could confuse translators.
